code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -207,7 +240,22 @@
 		if(3)
 			adjustBruteLoss(30)
 
-/mob/living/simple_animal/slaughter/laughter/proc/release_friends()
+/mob/living/simple_animal/slaughter/laughter/refresh_consumed_buff()
+	melee_damage_lower -= 0.5 // JAPES
+	melee_damage_upper += 1
```

Comment:
is there a reason the slaughter and laughter damage changes are done differently